### PR TITLE
[10.0] [ADD]  change_management_customer_ref

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Change Management Customer Ref
===============================

Creates the customer reference and Change code in Change Management

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: change_management_customer_ref/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```
Comment (AaronHForgeFlow) [inline change_management_customer_ref/__init__.py]: Hi @nikul-serpentcs  Those headers are also reserved for the authors or people who make significant contributions (e.g. adding new features). For migrations adding Serpent as contributor is enough in this case. Can you leave the header as it was here and the other Pull Requests? You can add yourself as contributor in the readme.

Regards

**Review by AaronHForgeFlow — APPROVED** [review]

File: change_management_customer_ref/__init__.py (hunk 1 above)
Comment (nikul-serpentcs) [reply change_management_customer_ref/__init__.py]: @aheficent Sorry I was a bit confused in authors and Copyrights section, I thought we need to add ourself in Copyrights as we have contributed, but now it is clear, thanks!

I'll do the needful.

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Pull request merged**